File: tests/expected/lambda.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

show() {
  var myfunc = (x, y) => (x + y);
  print(sprintf("%s", [myfunc(1, 2)]));
}

main(List<String> argv) {
  show();
}

File: tests/expected/lambda.kt


fun show() {
    var myfunc: (Int, Int) -> Int = { x, y -> (x + y) }
    if (true) {
        val __tmp1 = myfunc(1, 2)
        println("$__tmp1")
    }
}

fun main(argv: Array<String>) {
    show()
}
